plugins/webp-uploads/helper.php

```diff
@@ -384,3 +384,14 @@ function webp_uploads_get_image_output_format(): string {
 function webp_uploads_sanitize_image_format( string $image_format ): string {
 	return in_array( $image_format, array( 'webp', 'avif' ), true ) ? $image_format : 'webp';
 }
+
+/**
+ * Helper function to check if the `webp_uploads_picture_element_enabled` option is enabled.
+ *
+ * @since n.e.x.t
+ *
+ * @return bool True if the option is enabled, false otherwise.
+ */
+function webp_uploads_picture_element_enabled(): bool {
```

Comment:
Would the inclusion of `_is_` be warranted?

```suggestion
function webp_uploads_is_picture_element_enabled(): bool {
```

Reply:
sure... renamed in d2af18444791163f2ec711428c95466be971b133